src/tiles.ts

```diff
@@ -106,26 +106,31 @@ export function lonLatZoomToTile(lonlat: LonLat, zoom: Zoom, metatile = 1, refer
   return geoCoordsToTile(lonlat, zoom, metatile, referenceTileGrid);
 }
 
-export function tileToBoundingBox(tile: Tile, referenceTileGrid: TileGrid = TILEGRID_WORLD_CRS84): BoundingBox {
+export function tileToBoundingBox(tile: Tile, referenceTileGrid: TileGrid = TILEGRID_WORLD_CRS84, clamp = false): BoundingBox {
   validateTileGrid(referenceTileGrid);
   validateTile(tile, referenceTileGrid);
   const metatile = tile.metatile ?? 1;
 
   const width = tileWidth(tile.z, referenceTileGrid) * metatile;
   const height = tileHeight(tile.z, referenceTileGrid) * metatile;
```

Comment:
The name of the function tileWidth needs to include that fact that its width in "geographic" units and not px

Reply:
done